includes/french-guide/sonarr-cf-french-web-scoring-en.md

```diff
@@ -1,7 +1,7 @@
 ??? abstract "Language Profiles Related Custom Formats Scoring - [Click to show/hide]"
 
     | Custom Format                                                                                                                   | Trash ID                                                   |                              VOSTFR Profile Score                               |                              MULTi.VF Profile Score                               |                              MULTi.VO Profile Score                               |
-    |---------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------|:-------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------:|
+    | ------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------- | :-----------------------------------------------------------------------------: | :-------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------: |
```

Comment:
Why are we reformatting this table?

Reply:
My VS Code automatically adapts the table, but I don't see any particular problem? It's even better this way, isn't it?
I can revert this change if you want

Reply:
it's also how I learned to make tables in markdown